src/orion/core/cli/db/load.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+# pylint: disable=,protected-access
+"""
+Storage import tool
+===================
+
+Import database content from a file.
+
+"""
+import logging
+
+from orion.core.cli import base as cli
+from orion.core.io import experiment_builder
+from orion.core.worker.storage_backup import load_database
+from orion.storage.base import setup_storage
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+DESCRIPTION = "Import storage"
+
+
+def add_subparser(parser):
+    """Add the subparser that needs to be used for this command"""
+    load_parser = parser.add_parser("load", help=DESCRIPTION, description=DESCRIPTION)
+
+    cli.get_basic_args_group(load_parser)
+
+    load_parser.add_argument(
+        "file",
+        type=str,
+        help="File to import",
+    )
+
+    load_parser.add_argument(
+        "-r",
+        "--resolve",
+        type=str,
+        choices=("ignore", "overwrite", "bump"),
+        required=True,
+        help="Strategy to resolve conflicts: "
+        "'ignore', 'overwrite' or 'bump' "
+        "(bump version of imported experiment). "
+        "When overwriting, prior trials will be deleted.",
+    )
+
+    load_parser.set_defaults(func=main)
+
+    return load_parser
+
+
+def main(args):
+    """Script to import storage"""
+    config = experiment_builder.get_cmd_config(args)
+    storage = setup_storage(config.get("storage"))
```

Comment:
There should be a verification here that the storage is a Legacy storage, and ideally also that it is pickleddb. Otherwise the `logging.info` below will fail.

Reply:
Oh wait, that's the destination storage. Not the source one. Then the `main` will fail here if the destination is not Legacy. Should only print `storage` instead of `storage._db`.

Reply:
Ok!